modules/scheduler/scheduler/src/main/java/org/motechproject/scheduler/service/impl/MotechSchedulerServiceImpl.java

```diff
@@ -786,7 +797,7 @@ private void validateJob(JobKey key) throws SchedulerException {
 
         JobDataMap map = detail.getJobDataMap();
 
-        if (map != null && !map.getBooleanValue(UI_DEFINED)) {
+        if (map != null && map.get(EVENT_METADATA) != null && !(Boolean) ((Map<String, Object>) map.get(EVENT_METADATA)).get(UI_DEFINED)) {
```

Comment:
It is impossible to use getBooleanValue here, because (Map<String, Object>) map.get(EVENT_METADATA) map that we are using here is a HashMap, not a JobDataMap, so there isn't getBooleanValue implemented.